src/CaptureFileInfo/ManagerTest.cpp

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+
+#include <QCoreApplication>
+#include <QDateTime>
+#include <chrono>
+#include <thread>
+
+#include "CaptureFileInfo/Manager.h"
+#include "OrbitBase/ExecutablePath.h"
+
+namespace orbit_capture_file_info {
+
+constexpr const char* kOrgName = "The Orbit Authors";
+
+TEST(Manager, Clear) {
```

Comment:
I think we should call the test suite `CaptureFileInfoManager` instead of `Manager` because if it fails on the CI it's hard to figure out from the log that it refers to a failing test in `CaptureFileInfo` if it just says the test `Manager.Clear` failed.

Reply:
done